src/components/Firestore/Requests/RequestDetails/InspectionSection/index.tsx

```diff
@@ -33,7 +34,16 @@ const InspectionSection: React.FC<Props> = ({
     data-testid="request-details-inspection-section"
     className="Firestore-Request-Details-Inspection"
   >
-    Inspection Section
+    <InspectionBlock isMainBlock label="Query Information">
```

Comment:
We generally try to use sentence case, so in this case the label should be "Query information"